update.py

```diff
@@ -43,6 +43,20 @@ def main():
 
     print("Passed Git checks...")
 
+    # need to run using python 3.5 or higher to so update_deps can be running
+    # reffer to issue https://github.com/Just-Some-Bots/MusicBot/issues/1724
+
+    if not sys.version_info.major == 3:
+    	raise EnvironmentError("Please run this file using python 3.5.3 or higher ")
+
+    if sys.version_info.minor < 5:
+    	raise EnvironmentError("Please run this file using python 3.5.3 or higher ")
+
+    if sys.version_info.micro < 3:
+    	raise EnvironmentError("Please run this file using python 3.5.3 or higher ")
```

Comment:
Please change these 2 lines to
```
    if sys.version_info.minor == 5 and sys.version_info.micro < 3:
    	raise EnvironmentError("Please run this file using python 3.5.3 or higher ")
```
To avoid error in Python version higher than 3.5 but have micro version less than 3 (e.g. 3.7.0)